tech.json

```diff
@@ -727,7 +727,29 @@
         {
           "name": "canShinespark",
           "requires": [ "SpeedBooster" ],
-          "note": "The ability to use the Speed Booster to store a charge, then fly in a direction until an obstacle is hit."
+          "note": "The ability to use the Speed Booster to store a charge, then fly in a direction until an obstacle is hit.",
+          "extensionTechs": [
+            {
+              "name": "canMidairShinespark",
+              "requires": ["canShinespark"],
+              "note": "The ability to jump before shinesparking vertically, horizontally, or diagonally"
+            }
+          ]
+        },
+        {
+          "name": "canConserveShinesparkHealth",
```

Comment:
I think canConserveShinesparkHealth should be an extension of canMidairShinespark. So they are all nested. You can also remove canShinespark as a requires, since it's implied by canMidairShinespark